src/command/client/stats.rs

```diff
@@ -72,32 +68,23 @@ impl Cmd {
         settings: &Settings,
     ) -> Result<()> {
         let context = current_context();
-
-        match self {
-            Self::Day { words } => {
-                let words = if words.is_empty() {
-                    String::from("yesterday")
-                } else {
-                    words.join(" ")
-                };
-
+        let words = if self.period.is_empty() {
+            String::from("all")
+        } else {
+            self.period.join(" ")
+        };
+        let history: Vec<History>;
+        match words.as_str() {
+            "all" => {
+                history = db.list(FilterMode::Global, &context, None, false).await?;
+            }
```

Comment:
Nit: It's normally more rusty to do
```suggestion
        let history = match words.as_str() {
            "all" => {
                db.list(FilterMode::Global, &context, None, false).await?
            }
```
That is, a match statement is actually an expression and can produce values. Just omit the `;` at the end and that value will propagate out

Reply:
Ah, clippy is suggesting this too :)

Reply:
Missed this. Updating right now !